modules/backend/lang/fr/lang.php

```diff
@@ -344,10 +366,13 @@
         'permissions'  => 'PHP ne peut pas écrire dans le répertoire :name et ses sous-dossiers. Veuillez modifier les permissions en écriture du serveur web pour ce répertoire.',
         'extension' => 'L’extension PHP :name n’est pas installée. Veuillez installer la librairie et activer l’extension.',
         'plugin_missing' => 'Le plugin :name est une dépendance mais n\'est pas installé. Veuillez installer le plugin.',
+        'debug' => 'Le mode débogage est activé. Ceci n\'est pas recommandé pour les installations de production.',
```

Comment:
For strings with lots of escape sequences needed you can just wrap with double quotes instead to avoid having to escape all the single quotes.

Reply:
In this case a single one exists.  
I did it for line 23, for example.